INSTALL.md

```diff
@@ -126,7 +126,7 @@ The standard GNU/Linux tools and Python are minimal requirements to compile Shog
 To compile the interfaces, in addition to [swig](http://www.swig.org/) itself, you will need language specific development packages installed, see [interfaces](#manual-interfaces) below.
 
 There is a larger number of optional requirements.
-The output of cmake output lists optional dependencies that were found and not found.
```

Comment:
I think there is a different part of the document that mentions those? Maybe better put this there?
Also please put a new line for each sentence and respect whitespace.

Thanks!